tests/configuration/permissions_test.js

```diff
@@ -215,3 +195,40 @@ Data(ptSummaryRoleCheck).Scenario(
     I.seeElement(dashboardPage.nodeSummaryDashboard.ptSummaryDetail.reportContainer);
   },
 );
+
+Data(settingsReadOnly).Scenario(
+  'PMM-T1987 - Verify viewer/editor users can get settings/readonly @fb-settings',
```

Comment:
Should there also be a test, or it can be part of this test that viewer/editor cannot change settings?